src/broker/boot_config.c

```diff
@@ -522,6 +488,7 @@ int boot_config (flux_t *h, struct overlay *overlay, attr_t *attrs)
     json_t *hosts = NULL;
     struct topology *topo = NULL;
     flux_error_t error;
+    const char *topo_uri;
```

Comment:
Shouldn't the precedences be listed from highest to lowest b/c you list "1, 2, 3" next to them?

```
Add a new attribute: tbon.topo which can be set to a topology URI.
The value is determined by the following in ascending prcedence:
1. compiled-in default ('custom' for config bootstrap, 'kary:2' for PMI)
2. TOML configuration
3. command line
```

Reply:
I'll drop the numbers if that's confusing.